tests/test_gtfs_objects.py

```diff
@@ -0,0 +1,251 @@
+import datetime
+import os
+import pytest
+
+import numpy as np
+import pandas as pd
+
+from fasttrips import Route
+from fasttrips import Stop
+from fasttrips import Transfer
+from fasttrips import Trip
+
+HOME_DIR = os.path.join(os.getcwd(), "fasttrips", "Examples", )
+NETWORK_HOME_DIR = os.path.join(HOME_DIR, 'networks')
+
+
+@pytest.fixture(scope='module', params=['simple'])
+def scenario(request):
+    return request.param
+
+
+@pytest.fixture(scope="module")
+def network_dir(scenario):
+    yield os.path.join(NETWORK_HOME_DIR, scenario)
+
+
+@pytest.mark.skip(reason='Need to upgrade to Partridge')
+def test_stops_load(network_dir, gtfs_feed):
+    stops = Stop(network_dir, os.path.join(HOME_DIR, 'test_gtfs_objects'),
+                      gtfs_feed, datetime.date(2015, 11, 23))
+
+    #Test existence, length, and required columns
+    assert not stops.stop_id_df.empty
+    assert len(stops.stop_id_df) == 8
+    assert ({Stop.STOPS_COLUMN_STOP_ID, Stop.STOPS_COLUMN_STOP_ID_NUM}.issubset(stops.stop_id_df))
+    assert stops.max_stop_id_num == 8
+
+    #Test existence, length, dtype, and column names for stops_df
+    assert not stops.stops_df.empty
+    assert len(stops.stops_df == 8)
+    stop_df_dtypes = {
+
+        'location_type': np.int64,
+        Stop.STOPS_COLUMN_STOP_ID: object,
+        Stop.STOPS_COLUMN_STOP_LATITUDE: np.float64,
+        Stop.STOPS_COLUMN_STOP_LONGITUDE: np.float64,
+        Stop.STOPS_COLUMN_STOP_NAME: object,
+        Stop.STOPS_COLUMN_ZONE_ID: object,
+        Stop.STOPS_COLUMN_STOP_ID_NUM: np.int64,
+        Stop.STOPS_COLUMN_ZONE_ID_NUM: np.float64
+    }
+    assert (set(stop_df_dtypes.keys()).issubset(stops.stops_df))
+    assert stops.stops_df.dtypes.to_dict() == stop_df_dtypes
+    pd.testing.assert_frame_equal(stops.stop_id_df.set_index(Stop.STOPS_COLUMN_STOP_ID_NUM),
+                                  stops.stops_df.set_index(Stop.STOPS_COLUMN_STOP_ID_NUM),
+    )
+
+    #Test to make sure stop_id and stops dataframe are identical
+    stops_with_zones = stops.stops_df[stops.stops_df[Stop.STOPS_COLUMN_ZONE_ID_NUM].notnull()]
+    assert len(stops_with_zones) == 2
+    assert len(stops_with_zones[stops_with_zones[Stop.STOPS_COLUMN_STOP_ID].isin(['B1','B3'])]) == 2
+
+
+    assert len(stops.zone_id_df) == 2
+    pd.testing.assert_frame_equal(stops_with_zones[[Stop.STOPS_COLUMN_ZONE_ID_NUM, Stop.STOPS_COLUMN_ZONE_ID]].drop_duplicates().set_index(Stop.STOPS_COLUMN_ZONE_ID_NUM),
+        stops.zone_id_df.set_index(Stop.STOPS_COLUMN_ZONE_ID_NUM), check_dtype=False, check_index_type=False)
+
+    assert not stops.trip_times_df
+
+@pytest.mark.skip(reason='Need to upgrade to Partridge')
+def test_routes_load(network_dir, gtfs_feed):
```

Comment:
add description

Reply:
Cleaned up test_gtfs_objects to be compatible with Partridge. Added docstrings.